LayoutKitTests/TextViewLayoutTests.swift

```diff
@@ -0,0 +1,163 @@
+// Copyright 2016 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import XCTest
+@testable import LayoutKit
+
+class TextViewLayoutTests: XCTestCase {
+
+    private let defaultTextViewLayoutFont = UIFont.systemFont(ofSize: 12)
+
+    func testNeedsView() {
+        let layout = TextViewLayout(text: "hi").arrangement().makeViews()
+        XCTAssertNotNil(layout as? UITextView)
+    }
+
+    func testTextViewLayout() {
+        for textTestCase in Text.testCases {
+            let textView = UITextView()
+
+            switch textTestCase.text {
+            case .unattributed(let text):
+                textView.text = text
+                textView.font = textTestCase.font ?? defaultTextViewLayoutFont
+            case .attributed(let attributedText):
+                // If font is using default, applied default font to attributedString
+                if let font = textTestCase.font {
+                    textView.attributedText = attributedText.createAttrbutedString(with: font)
+                } else {
+                    textView.attributedText = attributedText.createAttrbutedString(with: defaultTextViewLayoutFont)
+                }
+            }
+
+            textView.updateAttributes()
+
+            let layout = textTestCase.font.map({ (font: UIFont) -> Layout in
+                return TextViewLayout(text: textTestCase.text, font: font)
+            }) ?? TextViewLayout(text: textTestCase.text)
+
+            let layoutView = layout.arrangement()
+
+            // Skip if the both widths equal to 0, no need to check height
```

Comment:
Seems like we should return a non-zero height in this case to match UITextView intrinsicContentSize behavior.